kubernetes/tests/kubernetes/tests.default.bom

```diff
@@ -27,5 +27,6 @@ brooklyn.catalog:
           kubernetes.minRam: 2000
           kubernetes.minCores: 1
           autoscaler.resizeUpStabilizationDelay: 60s
+          kubernetes.scaling.cpu.limit: 2.0
```

Comment:
Is that also part of this PR? Not sure what it does

Reply:
Increases the threshold that will trigger the up scaling.  I don't know exactly what this represents but in general a high number seems to be about 1.2 so this should stop scaling.  While making the resizeUpStabalization delay is probably enough I couldn't really tell from just sensor info.